### PR TITLE
RethinkDB - Fixed UpdateOptions

### PR DESCRIPTION
**Pull request by joaojacome**
Please fill in this template.

- [X] Make your PR against the `master` branch.
- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).
- [X] Run `tsc` without errors.

Select one of these and delete the others:

If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: https://www.rethinkdb.com/api/javascript/update/

### CODE REVIEW

**Comment by alexgorbatchev** [comment]
👍

File: rethinkdb/index.d.ts [hunk 1]
```diff
@@ -310,9 +310,9 @@ declare module "rethinkdb" {
     }
 
     interface UpdateOptions {
-        non_atomic: boolean;
-        durability: string; // 'soft'
-        return_vals: boolean; // false
+        nonAtomic?: boolean | false;
```
Comment (aozgaa) [inline rethinkdb/index.d.ts]: Why `boolean | false`? This type is equivalent to `boolean`

File: rethinkdb/index.d.ts [hunk 2]
```diff
@@ -310,9 +310,9 @@ declare module "rethinkdb" {
...
+        durability?: 'hard' | 'soft';
+        returnChanges?: boolean | false;
```
Comment (aozgaa) [inline rethinkdb/index.d.ts]: ditto

Comment (joaojacome) [inline rethinkdb/index.d.ts]: That's an excellent question! Sadly, it's one I don't know the answer. Fixed.

**Pull request merged**